android-components/components/feature/addons/src/main/java/mozilla/components/feature/addons/amo/AMOAddonsProvider.kt

```diff
@@ -152,9 +154,12 @@ class AMOAddonsProvider(
     @Throws(IOException::class)
     override suspend fun getAddonsByGUIDs(
         guids: List<String>,
+        allowCache: Boolean,
         readTimeoutInSeconds: Long?,
         language: String?,
     ): List<Addon> {
+        // We should probably introduce some sort of caching to avoid calling the AMO API all the time.
```

Comment:
@Amejia481 I _think_ this will be needed eventually but I am not sure how we'll want to implement that... I don't think we necessarily need a cache file stored on disk, maybe we can have in-memory caching? I think we need caching because this method is called in `getAddons()`, which is invoked every time a user opens the Fenix add-ons manager. WDYT?